src/storage/storage.h

```diff
@@ -358,8 +365,49 @@ class Storage {
 
   rocksdb::WriteOptions default_write_opts_ = rocksdb::WriteOptions();
 
-  rocksdb::Status writeToDB(const rocksdb::WriteOptions &options, rocksdb::WriteBatch *updates);
+  rocksdb::Status writeToDB(engine::Context &ctx, const rocksdb::WriteOptions &options, rocksdb::WriteBatch *updates);
   void recordKeyspaceStat(const rocksdb::ColumnFamilyHandle *column_family, const rocksdb::Status &s);
 };
 
+/// Context passes fixed snapshot and batche between APIs
+/// Limitations: Performing a large number of writes on the same Context may reduce performance.
+/// Please choose to use the same Context or create a new Context based on the actual situation.
+/// Context does not provide thread safety guarantees and is generally only passed as a parameter between APIs.
```

Comment:
LGTM except the style
```suggestion
/// Context passes fixed snapshot and batche between APIs
///
/// Limitations: Performing a large number of writes on the same Context may reduce performance.
/// Please choose to use the same Context or create a new Context based on the actual situation.
///
/// Context does not provide thread safety guarantees and is generally only passed as a parameter between APIs.
```